packages/graphql_server/src/index.ts

```diff
@@ -1,3 +1,5 @@
 import app from './server'
+import { getCompanyInfosFromLinkedIn } from './LinkedInScrapingService'
```

Comment:
please delete the lines you added to index.ts

Put it into graphql_server/tests/LinkedInScrapingService.ts instead.